modules/power-monitoring-dashboards-overview.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-dashboards-overview_{context}"]
+= {PM-shortname-c} dashboards overview
+
+There are two types of {PM-shortname} dashboards. Both provide different levels of details around power consumption metrics for a single cluster:
+
+[discrete]
+== Power Monitoring / Overview dashboard
+
+With this dashboard you can observe the following information:
+
+* An aggregated view of CPU architecture and its power source (`rapl-sysfs`, `rapl-msr`, or `estimator`) along with total nodes with this configuration
+
+* Total energy consumption by a cluster in the last 24 hours (measured in kilowatt-hour)
+
+* The amount of power consumed by the top 10 namespaces in a cluster in the last 24 hours
+
+* Detailed node information, such as its CPU architecture and component power source
+
+These features allow you to effectively monitor the energy consumption of the cluster without needing to investigate each namespace separately.
+
+[WARNING]
+====
+Ensure that the *Components Source* column does not display `estimator` as the power source.
+
```

Comment:
It would be good if we could list which architectures are supported.

Reply:
How would such a list look like?

The statement is intentionally generic because as long as `estimator` is not used, kepler does not rely on ML models to predict the power consumption. Rather, it relies on the information exposed by kernel through sysfs - see: https://docs.kernel.org/power/index.html  or other means. This information can be relied upon.

Reply:
hello @stleerh @sthaha This is a bigger change that I am unable to add right before the release after the docs freeze (it would also definitely need additional reviews from peer and QE). If this is something you would like to be added, the doc  Jira issue needs to be created and the architectures will be added after the release. Thank you for understanding!

Reply:
Oh I just checked, I am wrong about that last one :sweat_smile: But my worry is still the same.

Reply:
@ekristova , I am happy with this content as is since I think listing all cpu's supported by RAPL is not be feasible.

Reply:
As a start, what bare metal servers have we tested?

Reply:
@stleerh like, @ekristova , pointed out ... 
> I am unable to add right before the release after the docs freeze 

Lets use jira to continue our discussion on whether or not this information is relevant on a user-guide. It would be nice if you could point out any precedence to this in our docs so that we can use that as a reference. 

I am going to resolve this and continue the discussion on a jira.